vmdk_plugin/vmdkops/esx_vmdkcmd.go

```diff
@@ -93,6 +93,9 @@ func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) (
 		msg := fmt.Sprintf("'%s' failed: %v (errno=%d).", cmd, err, int(errno))
 		if errno == syscall.ECONNRESET {
 			msg += " Check that ESX service is running."
```

Comment:
Rather than embedding these error messages here, how about introducing separate init-handshake op. Tools/VMCI/Port error messages doesnt make sense after first command is completely successfully.